### DISCUSSION TITLE
See the number of interractions

### DISCUSSION THREAD

**Discussion by Noe-Briffa** [FlowiseAI/Flowise#804] [Q&A]
Hello,

I would like to know if it is possible to see the number of interactions or words a ChatBot receives?

Thanks in advance!

**Comment by chungyau97** [comment]
Hi @Noe-Briffa,

This will be related to PR #715.

**Comment by Noe-Briffa** [comment]
Thank you for your reply!
In fact I don't think I fully understood how it works. Is it possible to have a chat history using Flowise's "embed" feature for the ChatBot?
Thank you.

**Reply by chungyau97** [reply]
Yes, this PR will allow "embed" to have history/
